### PR TITLE
add guide about blocks in runtime

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

- Add guide about blocks and a short example how it could be used in test

## TODO

- [ ] update change log

### CODE REVIEW

File: docs/guide/testing-teal.md [hunk 1]
```diff
@@ -25,16 +25,25 @@ The `@algo-builder/runtime` (TypeScript Algorand runtime) package has 4 major co
...
 
-This means that current block round is set to 5 and transaction will pass only if its first valid round is less or equal 5 and the last valid round is greater than 5.
-Note: Block round and timestamp remains same until user will change it again.
+The default Runtime block round is set to `2000`.
+
+Example: 
+```typescript
+const runtime = new Runtime();
+//some tests
```
Comment (rmeena840) [inline docs/guide/testing-teal.md:38]: `//some tests` ???
Can you explain more in the comment, what it is?

Comment (rmeena840) [reply docs/guide/testing-teal.md:38]: pls do this for all comments here.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviewed

File: docs/guide/testing-teal.md (hunk 1 above)
Comment (sczembor) [inline docs/guide/testing-teal.md:38]:
```suggestion
//your tests
```

File: docs/guide/testing-teal.md [hunk 2]
```diff
@@ -25,16 +25,25 @@ The `@algo-builder/runtime` (TypeScript Algorand runtime) package has 4 major co
...
+runtime.produceBlocks(); //creates a new block
+//some more tests
```
Comment (sczembor) [inline docs/guide/testing-teal.md:40]:
```suggestion
//your tests
```

File: docs/guide/testing-teal.md [hunk 3]
```diff
@@ -25,16 +25,25 @@ The `@algo-builder/runtime` (TypeScript Algorand runtime) package has 4 major co
...
+runtime.getBlock(1900);
+//some more tests
```
Comment (sczembor) [inline docs/guide/testing-teal.md:42]:
```suggestion
//your tests
```

File: docs/guide/testing-teal.md [hunk 4]
```diff
@@ -25,16 +25,25 @@ The `@algo-builder/runtime` (TypeScript Algorand runtime) package has 4 major co
...
+runtime.produceBlocks(20); //creates 20 new blocks
+//some more tests
```
Comment (sczembor) [inline docs/guide/testing-teal.md:44]:
```suggestion
//your tests
```

**Review by rmeena840 — APPROVED** [review]
LGTM

**Pull request merged**